src/scripts/main.js

```diff
@@ -1,3 +1,255 @@
+    }
+  } else {
+    const p = document.createElement('p');
+
+    p.innerText = successDescription;
+    div.append(p);
+  }
+
+  div.classList.add(type);
+  div.style.top = '450px';
+
+  setTimeout(() => {
+    document.querySelectorAll('.notification').forEach(x => x.remove());
+  }, 6000);
+};
+
+// Sorting tables FUCTION
+function sortElemInTwoWays(element, start, end) {
+  if (element === 'Salary') {
+    const parceSalary = (salary) => {
+      return parseFloat(salary.slice(1).replace(',', '.'));
+    };
+
+    return parceSalary(start) - parceSalary(end);
+  } else {
+    return start.localeCompare(end);
+  }
+};
+
+// Submit form EVENT
+form.addEventListener('submit', e => {
+  e.preventDefault();
+
+  const data = new FormData(form);
+  const formData = Object.fromEntries(data.entries());
+  const salary = +formData.salary;
+  const message = [];
+  let type = 'success';
+
+  const formBoolean = {
+    name: formData.name.length >= 4,
+    position: formData.position.length >= 4,
+    office: formData.office !== '',
+    age: +formData.age >= 18 && +formData.age <= 90,
+    salary: +formData.salary > 90,
+  };
+
+  for (const check in formBoolean) {
+    if (!formBoolean[check]) {
+      type = 'error';
+      message.push(check);
+    }
+  }
+
+  pushNotification(message, type);
+
+  if (type === 'success') {
+    form.reset();
+
+    tbody.insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>${formData.name}</td>
+      <td>${formData.position}</td>
+      <td>${formData.office}</td>
+      <td>${formData.age}</td>
+      <td>$${salary.toLocaleString('en-US')}</td>
+    </tr>
+  `);
+  }
+});
+
+// Sorting table EVENT
+thead.addEventListener('click', e => {
+  const addClass = e.target.toggleAttribute('sorted');
+  const headerRow = thead.firstElementChild.children;
+  const index = [...headerRow].findIndex(elem => elem === e.target);
+
+  const sortedList = [...rows].sort((a, b) => {
+    const start = a.children[index].innerText;
+    const end = b.children[index].innerText;
+
+    if (addClass) {
+      return sortElemInTwoWays(e.target.innerText, start, end);
+    } else {
+      return sortElemInTwoWays(e.target.innerText, end, start);
+    }
+  });
+
+  sortedList.forEach(elem => tbody.append(elem));
+});
+
+// Selecting row EVENT
+tbody.addEventListener('click', e => {
+  for (const x of rows) {
+    if (x.classList.contains('active') && x !== e.target.parentElement) {
+      x.classList.remove('active');
+    }
+  }
+
+  e.target.parentElement.classList.toggle('active');
+});
+
+// Eit cell EVENT
+tbody.addEventListener('dblclick', e => {
+  const initialText = e.target.innerText;
+  const target = e.target.parentElement;
+  const input = document.createElement('input');
+  const select = document.createElement('select');
+  const theadTitle = thead.firstElementChild.children;
+  const index = [...target.children].findIndex(x => x === e.target);
+  const styleTD = getComputedStyle(e.target);
+  let inputOrSelect = true;
```

Comment:
Consider using a more descriptive name for the 'inputOrSelect' variable, like 'isInput' or 'useInput'.